### ISSUE TITLE
Set up demo for App's session authorization

### ISSUE THREAD

**Issue by tyland5** [cse442-at-ub/project_s23-cinco#56]
*Test Cases*

Local host with xampp apache running

*Test 1*
1) After having the php file in the htdocs of the xampp folder on your pc and with xamp running apache, put in the local host url into your browser. Similar to what appears in the image right below in step 2
2) Inspect the page and go to the application section. Under cookies, you should see this. The persistent cookie is named session. ![Screenshot (1012).png](https://images.zenhubusercontent.com/63e2cfbfaca3b22752e618ee/e99483ea-f9af-425d-bb97-db7cd5e168f6)

3) Then go to phpmyadmin and check if there are changes made to the session id of user with id 1. For the purpose of testing, the token's expiration is a minute. So after a minute and closing out the browser (to change the session_id created by the php and make the change more noticeable in phpmyadmin), you can go to the page and see the changes in both the cookies and phpmyadmin. The database would look like this![Screenshot (1013).png](https://images.zenhubusercontent.com/63e2cfbfaca3b22752e618ee/5ed48b5c-84de-4952-87be-df6a91600d3e)

**Comment by tyland5** [comment]
Been doing research last night (3/13) and today (3/14) on what exactly JWT is and why we could use that instead of session/cookies. I learned that it helps with authentication between different webservers that a certain website may utilize and does not require storage of session ids in a database. This makes JWT "lightweight" and possibly easier to use. The webserver verifies controlled requests through hashing/signature

**Comment by tyland5** [comment]
Spent around 2-3 hours trying to find a good tutorial, especially with a php backend but couldnt

**Comment by tyland5** [comment]
Put in another 3 hours trying to figure how to create session ids and put it into cookies. Found out php session_start() pretty much creates a long non-clashable session id and puts it into your cookies. But, it's only temporary, meaning closing out the browser will clear it. Determining if I want to have permanent cookies that have an expiration date and won't expire upon closing the browser. However, doing so will require me to implement an interface and manage connections in the database. Will do more research 

Below is some experimentation I did. Generated a session id and set it to a cookie variable that expires in a day
[Uploading... Screenshot (1009).png]
